Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -6,6 +6,17 @@ import format from 'pg-format';
 
 // SQL for Selects
 
+/**
+ * Function that returns a SQL query for checking whether references exist for a given list of document ids
+ * @param documentIds the list of documents to check for existence
+ * @returns SQL query to check for existence ids
+ */
+export const existenceIdsForDocument = (existenceId: string): string =>
```

Comment:
should be named "documentId", also check @param

Reply:
updated